crowbar_framework/app/models/node_object.rb

```diff
@@ -1321,12 +1321,57 @@ def boot_device!(device)
     boot_device(device) and save
   end
 
+  # TODO: Remove duplicate code and use a gem/git submodule/whatever...
+  # see barclamp-deployer/chef/cookbooks/barclamp/libraries/barclamp_library.rb
+  def unique_name_already_claimed_by(device)
+    claimed_disks = @node[:crowbar_wall][:claimed_disks].to_hash
+    claimed_disks.each do | claimed_name, v |
```

Comment:
> If [:claimed_disks] value is an array of pairs, you can skip the conversion to hash, then use claimed_disk.find { |claimed_name, v| ... } instead of each + return. It will also neatly return nil, if such element does not exist, making this method a one- or -two liner.

But I don't want to have the pair as return value. I just want ```claimed_name```.